src/api/app/views/webui2/webui/user/_breadcrumb_items.html.haml

```diff
@@ -3,6 +3,10 @@
   = render partial: 'webui/configuration/breadcrumb_items'
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
     Manage Users
+- elsif current_page?(user_register_user_path)
+  = render partial: 'webui/configuration/breadcrumb_items'
+  %li.breadcrumb-item.active{ 'aria-current' => 'page' }
+    Create new user
```

Comment:
`new` is redundant here, let's remove it. It should also be capitalized, so: Create **U**ser